app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,76 @@
+<div class="flex justify-around my-9">
+  <div class="ml-9">
+    <%= link_to recipes_path do %>
+      <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+        <path stroke-linecap="round" stroke-linejoin="round" d="M10.5 19.5L3 12m0 0l7.5-7.5M3 12h18" />
+      </svg>
+    <% end %>
+  </div>
+  <div class="Recipe-details flex flex-col">
+    <% @recipe.each do |recipe| %>
+      <div class="font-bold text-3xl text-center mb-8"><%= recipe.name %></div>
+        <div class="flex gap-x-32">
+          <div>
+            <hr>
+            <div class="text-xl my-3 flex justify-between"><span class="text-gray-400">Preparation Time:</span> <%= recipe.preparation_time%></div> 
+                <hr>
+            <div class="text-xl my-3 flex justify-between"><span class="text-gray-400">Cooking Time:</span> <%= recipe.cooking_time%></div> 
+                <hr>
+            <div class="text-xl my-3 flex justify-between"><span class="text-gray-400">Descriptions:</span> <%= truncate(simple_format(word_wrap(recipe.description, line_width:40, break_sequence: '<br>')), {length: 70}) %></div> 
+            <hr>
+          </div>
+          <div class="flex gap-x-8">
+            <span class="font-bold">Public:</span>
+            <span><%= recipe.public.to_s.capitalize %></span>
+          </div>
+
+        </div>
+        <div class="action-button flex justify-between">
+          <button data-modal-target="popup-modal" data-modal-toggle="popup-modal" class="bg-gradient-base border text-center bg-blue-600 hover:bg-white text-white hover:text-blue-600 font-bold uppercase text-xs mt-5 py-4 px-6 rounded cursor-pointer" type="button">
+            Generate Shopping List
+          </button>
+            <!-- Modal toggle -->
+          <button data-modal-target="authentication-modal" data-modal-toggle="authentication-modal" class="bg-gradient-base border text-center bg-blue-600 hover:bg-white text-white hover:text-blue-600 font-bold uppercase text-xs mt-5 py-4 px-6 rounded cursor-pointer" type="button">
+            Add Ingredient
+          </button>
+          <%= render "modal", recipe: recipe %> 
+        </div>
```

Comment:
- [ ] Hello, the requirements says that On the recipe show page, you should be able to generate a shopping list that matches the given. Can you follow the requirements to fix it?
- Your App:
![11](https://github.com/fezzopro/recipe_app/assets/63972670/91c9128a-e3ff-4ddf-ba75-fa0e0679224c)
- Now trying to generate:
![1](https://github.com/fezzopro/recipe_app/assets/63972670/a5488151-d792-4d20-ae3b-665b974f7a20)
- Given design

![3](https://github.com/fezzopro/recipe_app/assets/63972670/ae6a7f8f-3a26-4629-8995-6319e71200c5)

Reply:
This change has been done well